css/main.css

```diff
@@ -325,3 +325,122 @@ li {
 .skills {
   list-style: inside url("../media/svg/circle.svg");
 }
+
+/* ### Footer ### */
+footer {
+  display: grid;
+  align-items: center;
+  padding: 80px 0 2px 0;
+}
+
```

Comment:
## Changes Required!

⚠️ You have styled your footer using CSS Grid. While it's a great tool, this project specifically requires you to style your contact form using CSS Flexbox. The Desktop version currently looks the same way as the mobile version, but in the Figma template, you are required to make a different layout - with text to the left side of your form.  Kindly use your ✨ CSS Flexbox ✨ powers to correct it.  💡 Hint: This is the best online resource for CSS Flexbox: https://css-tricks.com/snippets/css/a-guide-to-flexbox/ .

This is your project: 
![form-screenshot](https://user-images.githubusercontent.com/53036416/163649146-ed752c0b-c90f-4554-8c9c-6f8e68f332cf.png)

What we need to make it look like on Desktop:
![correct-form-screenshot](https://user-images.githubusercontent.com/53036416/163649153-225af130-8f5a-4ce5-88cc-a93ed6f17e9f.png)